app.js

```diff
@@ -0,0 +1,44 @@
+const { request } = require("http");
+
+class App {
+  commands = [];
+
+  registerCommand(name, callback) {
+    const object = { name: name, callback: callback };
+    this.commands.push(object);
+  }
+
+  showCommands() {
+    const names = this.commands.map((item) => item.name);
```

Comment:
Так как `names` используется исключительно в виде имен команд через запятую, имеет смысл сразу приготовить все данные
```suggestion
    const names = this.commands.map((item) => item.name)..join(", ");
```